docs/userguide/configuring.rst

```diff
@@ -1,17 +1,17 @@
 .. _configuration-section:
 
-Configuration
+Environment Configuration
```

Comment:
Environment is so overloaded that I'd rather not use it here.

Reply:
But configuration by itself is not enough.  Would execution be better?

Reply:
This section deals with Parsl configuration, and that's more than execution. We deal with data management, aspects of how to connect/authenticate, monitoring of tasks, logging. Parsl configuration is redundant, so I'd prefer to leave this as "Configuration".

Reply:
Can we say anything more specific than configuration?  I feel like this is too general

Reply:
@yadudoc - If you don't think there's anything better we can say, then l'll drop this part, but let's merge this PR soon ...

Reply:
drop it out of the PR and we can merge the rest now and argue about configuration phrasing in a different venue/PR

Reply:
ok, it's out.